tests/index_tests.cpp

```diff
@@ -101,9 +96,9 @@ TEST_CASE("Compound index") {
     };
     auto table = make_table("users", make_column("id", &User::id), make_column("name", &User::name));
 
-    using storage_impl_t = internal::storage_impl<decltype(table)>;
-    auto storageImpl = storage_impl_t{table};
-    using context_t = internal::serializer_context<storage_impl_t>;
+    using db_objects_t = internal::db_objects_tuple<decltype(table)>;
+    auto storageImpl = db_objects_t{table};
```

Comment:
nit: better rename `storageImpl` to `dbObjects` or something related to changed things

Reply:
an oversight on my part!